assignments/callbacks.js

```diff
@@ -25,22 +25,47 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 */
 
 
-function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+
+
+function getLength(arr, cb) { //this is the higher order function
+   return cb(arr);
 }
+const arrayLength = function (arr){ //this is the callback function
+  return (arr.length);
+};
+console.log(getLength(items, arrayLength)); //here is where i invoke the actual thing
+
+
+
+
+
+
+
 
-function last(arr, cb) {
-  // last passes the last item of the array into the callback.
+function last(arr, cb) { //this is the HOF (higher order function)
+  return cb(arr[arr.length-1]);
 }
 
-function sumNums(x, y, cb) {
-  // sumNums adds two numbers (x, y) and passes the result to the callback.
+
+last(items, function(last){ //this is the cb
```

Comment:
I love that you're putting comments everywhere to note where the CB and HOFs are